content/getting-started/sections/installing-atom.asciidoc

```diff
@@ -66,6 +66,11 @@ Note that there are some limitations to portable mode:
 * The .atom folder must be writeable
 * The ATOM_HOME environment variable must not be set (this overrides portable behavior)
 
+===== MSI installer for Windows
+Atom is also available in an .MSI package from https://github.com/atom/atom/releases/latest[Atom Releases] which can be deployed or installed on a machine. Any user signing in to that machine will find Atom installed for them automatically (it may take a few seconds the first time they sign in for the icon to appear on the desktop).
+
+Should the user uninstall Atom it will only affect themselves and it will not automatically reinstall when they next sign in unless the %LOCALAPPDATA%\Atom folder is deleted.
```

Comment:
If I'm understanding this correctly, if someone uninstalls Atom after being installed by the MSI system, it won't prompt them to reinstall when they next log in. If they then want to reinstall, they can delete the `%LOCALAPPDATA%\Atom` folder?

If that is correct, I'd recommend rewriting this line to be more clear. Perhaps like this:

> Should the user uninstall Atom it will only affect themselves and it will not automatically reinstall when they next sign in. If eventually that individual wants to reinstall Atom again, they can delete the %LOCALAPPDATA%\Atom folder.